grass7/imagery/i.modis/i.modis.download/i.modis.download.py

```diff
@@ -195,6 +199,8 @@ def main():
                           "the username and password"))
     # set username, password and folder by file
     else:
+        if os.path.isdir(options['settings']):
```

Comment:
I'm not an expert, but I wonder if `isdir()` will indeed test if a path is a file? I also found that there's `isfile()` (https://docs.python.org/3/library/os.path.html). I think isdir() will always be false, either the file in the path exists or not, at least that's what my toy tests show here.

Reply:
I agree

Reply:
Fixed by Anika